### PR TITLE
Added -PskipDmgSign param for dmg task, readme.md updated/refreshed

### PR DESCRIPTION
**Pull request by pskowronek**
- Added -PskipDmgSign param for `dmg` task (for dev purposes) + readme.md updated for it
- readme.md: 
  - removed trailing spaces
  - removed Zanata (since it doesn't work no more), replaced with Translation wiki page (which should be updated to remove Zanata from there also)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,27 @@
...
 ## Contribution
 
 
-There are several ways to contribute to muCommander:  
+There are several ways to contribute to muCommander:
 
 - Found a bug or thinking about a useful feature that is missing? [File an issue](https://github.com/mucommander/mucommander/issues)
 - Want to fix a bug or implement a feature? We are using the standard [GitHub flow](https://guides.github.com/introduction/flow/): fork, make the changes, and submit a pull request. Changes are merged to the *master* branch. See the next section for tips for developing muCommander.
-- If you happen to speak a language that muCommander is not available in or able to improve existing translations, you can help translate the interface using the [zanata platform](https://translate.zanata.org/project/view/mucommander).  
+- If you happen to speak a language that muCommander is not available in or able to improve existing translations, you can help translate the interface see [here](https://github.com/mucommander/mucommander/wiki/Translate) for more details.
```
Comment (ahadas) [inline README.md:21]:
```suggestion
- If you happen to speak a language that muCommander is not available in or able to improve existing translations, you can help translate the interface, see more details [here](https://github.com/mucommander/mucommander/wiki/Translate).
```

File: README.md [hunk 2]
```diff
@@ -35,50 +35,56 @@ If you want to get involved in muCommander or have any question or issue to disc
...
+```
+Then, you can run a debugger that connects to this port using your favorite IDE (see [an example for doing this with IntelliJ](https://github.com/mucommander/mucommander/wiki/Debug-from-IntelliJ)).
 
 
 ### Packaging
-The creation of a DMG file for MAC OS (produced in build/distributions):  
+The creation of a DMG file for macOS (produced in build/distributions):
 ```
-./gradlew dmg
+./gradlew clean dmg -PskipDmgSign
```
Comment (ahadas) [inline README.md:69]: but the `clean` task is not required for this, why adding it here and below?

**Review by ahadas — COMMENTED** [review]
looks good overall

File: README.md (hunk 2 above)
Comment (pskowronek) [reply README.md:69]: I have impression that changes may not be properly included in DMG (if bundles were built before the change). And, additionally, `jpackage` complains that .app already exists:
```
build/muCommander.app already exists
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.createRoot(AppImageBundler.java:155)
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.createAppBundle(AppImageBundler.java:174)
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.execute(AppImageBundler.java:91)
        at jdk.jpackage/jdk.jpackage.internal.Arguments.generateBundle(Arguments.java:700)
        at jdk.jpackage/jdk.jpackage.internal.Arguments.processArguments(Arguments.java:565)
        at jdk.jpackage/jdk.jpackage.main.Main.execute(Main.java:92)
        at jdk.jpackage/jdk.jpackage.main.Main.main(Main.java:53)
```

But the real reason was to be sure that recent changes are there.

Comment (ahadas) [reply README.md:69]: okok

**Review by ahadas — APPROVED** [review]

**Pull request merged**